app/views/procurements/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<nav>
+  <div>
+    <%= link_to '<', categories_path %>
+  </div>
+  <div>
+    <h2>Transactions</h2>
+  </div>
+  <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+</nav>
+<section class="main-category">
+  <div class="category-box">
+    <img src="<%= @category.icon %>">
+    <p><%= @category.name %></p>
+  </div>
+  <% procurement_sum = 0 %>
+  <% @procurements.each do |procurement| %>
+    <% procurement_sum += procurement.amount %>
+  <% end %>
+  <div>
+    <p>Total: $<%= procurement_sum %></p>
+  </div>
+</section>
+<section class="main">
+  <% @procurements.each do |procurement| %>
+  <section class="procurement">
+    <div>
+      <p><%= procurement.name %></p>
+      <p><%= procurement.updated_at.strftime("%e %b %Y") %></p>
+    </div>
+    <div>
+      <p>$<%= procurement.amount %></p>
+    </div>
+  </section>
+  <% end %>
+</section>
+<div class="add-btn btn">
+  <%= link_to 'Add a New Transaction', new_category_procurements_path %>
+</div>
```

Comment:
| @Wineshuga Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/4f302615-e46e-469c-9b6a-e1ff252614c2) |


| Microverse Design |
|-|
| ![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/32db909c-460a-49b4-b556-653ab4a9feac) |

- [ ] It would be nice if you made your button look similar to the given design because the white border almost touches the edges of the green borders 🤔 to make your design look professional 👍 Your white border is touching the upper and lower edges so you just need to make it touch the left and right edges 😊